test/integration/emoji.test.js

```diff
@@ -150,7 +151,8 @@ describe('Emoji', function () {
   test('Ignores emoji shorthand codes in style url() values', async () => {
     await docsifyInit({
       markdown: {
-        homepage: /* html */ `<style>@import url(http://domain.com/:smile/file.css);</style>`,
+        homepage:
+          /* html */ '<style>@import url(http://domain.com/:smile/file.css);</style>',
```

Comment:
Did we shorten our allowable line length? Would prefer not to do that.

Reply:
Nope. Line length was not changed. My guess is that this due to a change in how Prettier handles comments in v3.